com.github.leeroymanea.synology-note-station.yaml

```diff
@@ -0,0 +1,40 @@
+# copied/inspired from https://github.com/flathub/com.synology.SynologyDrive/blob/master/com.synology.SynologyDrive.yaml
+id: com.github.leeroymanea.synology-note-station
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+arch:
+  - x86_64
+tags:
+  - proprietary
+command: /app/opt/synology-note-station/launch.sh
+finish-args:
+  - --filesystem=home
```

Comment:
why do you need home?

Reply:
the launch.sh from synology is accessing the home folder. 

```bash
#!/bin/sh

NS_DIR="$HOME/.syno_ns_app"
NS_DIR_BACKUP="$HOME/.syno_ns_app.bak"
NS_DATA_DIR=".note_data"
NS_BIN="$NS_DIR/synology-note-station"

if [ ! -x "$NS_BIN" ]; then
	mkdir -p $NS_DIR/
	cp -af /opt/synology-note-station/* $NS_DIR/
fi

if [ ! -d "$NS_DIR/$NS_DATA_DIR" -a -d "$NS_DIR_BACKUP" -a -d "$NS_DIR_BACKUP/$NS_DATA_DIR" ]; then
	cp -af "$NS_DIR_BACKUP/$NS_DATA_DIR" "$NS_DIR/$NS_DATA_DIR"
	rm -rf "$NS_DIR_BACKUP"
fi

cd $NS_DIR
exec $NS_BIN
```

Did i something wrong?

Reply:
Try using `--persist=.syno_ns_app`, should not need home access anymore.

Reply:
@bbhtt is not working for me. 
```
  - --persist=.syno_ns_app
  - --persist=.note_data
```
i have added both folders from the launch.sh script 
```
cp: der Aufruf von stat für '/opt/synology-note-station/*'
```
file or path not found

Reply:
`/opt/synology-note-station/` does not exist it is under `/app/opt`.

Reply:
Also why do you need to copy files there?

Reply:
@bbhtt i would like to leav it how it is. 
I dont want to change the code "launch.sh" from Synology to get it to work.  Because i dont know the side effects.

Reply:
@bbhtt 
```
Also why do you need to copy files there at runtime?
```
As i said.  The code is from Synology and not from me. 

I belief this is used in note stations update functionality.

As far as my tests go. With the current state is everything working for me.

Reply:
@bbhtt 
i have tried it without the "home" folder permission and without the "launch.sh" 
The application works. But it does not remember the User session. You have to log in again every time you start the application. 

But if the home permission is an issue i preventing the app to get published , i would remove the home  completly. 

Stil i would like to remain the permission. 
Personally i would not use it, if i would have to type in every time the synology nas address, username and password. 

What should I / We do now?

Reply:
you need to figure out where it write stuff. Or you can modify the launch.sh.

Reply:
is solved now.